src/StoreReview.Plugin/StoreReviewImplementation.apple.cs

```diff
@@ -79,6 +80,15 @@ public Task RequestReview(bool testMode)
 #if __IOS__
             if (IsiOS103)
             {
+				if (IsiOS14)
+				{
+					var windowScene = UIApplication.SharedApplication.ConnectedScenes.ToArray<UIScene>()?.First(x => x.ActivationState == UISceneActivationState.ForegroundActive) as UIWindowScene;
```

Comment:
Any additional null checks here? like on 

UIApplication.SharedApplication?.ConnectedScenes?.ToArray<UIScene>()?.First

Also shoudl we use FirstOrDefault?... if there are none then First would throw.

Reply:
@jamesmontemagno good catches! Updated the PR, Tested that and it works!